### ISSUE TITLE
2 code errors in Ex. 3 of ARIA2: Identifying a required field with the aria-required property

### ISSUE THREAD

**Issue by dmargaretosIBM** [w3c/wcag#90]
ARIA2: Identifying a required field with the aria-required property. http://www.w3.org/TR/WCAG20-TECHS/ARIA2.html

Code error 1:  The 3rd example shows individual radio button with the aria-required property. I believe this is incorrect. According to the ARIA spec ( http://www.w3.org/WAI/PF/aria/states_and_properties#aria-required), "aria-required" is not supported on role="radio" (it should be on the element with role="radiogroup"). 
Code error 2: Also noticed that the id is left off of the <input> element.  

Existing and corrected code samples provided:

Existing code sample for Example 3:
<head>
<form action="#" method="post" id="alerts1">
  <label for="acctnum" data-required="true">Account Number</label>
  <input size="12" type="text"
      aria-required="true" name="acctnum" />

 <p id="radio_label">Please send an alert when balance exceeds $3,000.</p>

 <ul  id="rg" role="radiogroup" aria-labelledby="radio_label">
    <li id="rb1" role="radio" aria-required="true">Yes</li>
    <li id="rb2" role="radio" aria-required="true">No</li>
 </ul>
</form>

Here is how I think Example 3 should be constructed ("id" added to the input element; moved "aria-required" to radiogroup):
<head>
<form action="#" method="post" id="alerts1">
  <label for="acctnum" data-required="true">Account Number</label>
  <input size="12" type="text"
      aria-required="true" name="acctnum" id="acctnum"/>

 <p id="radio_label">Please send an alert when balance exceeds $3,000.</p>

 <ul  id="rg" role="radiogroup" aria-labelledby="radio_label" aria-required="true">
    <li id="rb1" role="radio">Yes</li>
    <li id="rb2" role="radio">No</li>
 </ul>
</form>

end

**Comment by jnurthen** [comment]
I agree about the aria-required. This should be moved to the radiogroup.

Input elements don't need an id so I see no need to add that to the input
element.

**Comment by awkawk** [comment]
Anyone want to do a pull request with the fixes?

**Comment by jnurthen** [comment]
Which branch?

**Comment by awkawk** [comment]
Working-Branch-for-Q3-2015

**Comment by jnurthen** [comment]
Can you (or should I) fix the readme in that branch which says
"
WCAG (Web Content Accessibility Guidelines) - Branch for production of Jan
2015 review versions

Please do not submit any new pull requests against this branch."

**Comment by awkawk** [comment]
Thanks for the heads up - I made the changes.

**Comment by dmargaretosIBM** [comment]
Actually, I do think we need the id on the <input> element:
1. The for attribute is supposed to use the element id that it's bound to:  http://www.w3schools.com/tags/att_label_for.asp
2. WCAG technique requires the use of the for and id attributes to explicitly associate the label with the form control:  http://www.w3.org/TR/WCAG20-TECHS/H44.html
Feel free to contact me if you want to discuss. 
Many thanks.

**Comment by dmargaretosIBM** [comment]
Sorry - didn't mean to close the issue.

**Comment by jnurthen** [comment]
you are right. I missed that is was being labelled using for... I'll fix
both but the code sample needs a bunch of changes for the styling now that
aria-required is being moved.

**Comment by jnurthen** [comment]
Oops I accidentally pushed this fix to the working branch. 
Andrew - can you let me know how I should proceed?

**Comment by awkawk** [comment]
James, since this was a small change and the group approved it with a couple of changes we didn't need to change the target for the pull request. 

This issue is resolved by these commits:
https://github.com/w3c/wcag/commit/4e7c715badacf6ff19717a01631e6b3f9986d549
https://github.com/w3c/wcag/commit/c7ae7bbcbe06a027f0540c24441c506b0c6bfe80
